Source/Examples/ExampleLibrary/Axes/AxisExamples.cs

```diff
@@ -51,6 +51,29 @@ public static PlotModel TickStyleOutside()
             return CreateTickStyleExample(TickStyle.Outside);
         }
 
+        [Example("TickStyle: Color major and minor ticks differently")]
+        public static PlotModel TickLineColor()
+        {
+            var plotModel1 = new PlotModel { Title = "Color major and minor ticks differently" };
+            plotModel1.Axes.Add(new LinearAxis
+            {
+                Position = AxisPosition.Left,
+                MajorGridlineThickness = 3,
+                MinorGridlineThickness = 3,
+                TicklineColor = OxyColors.Blue,
```

Comment:
To me either makes sense. Prefixing with Major keeps consistent naming. But TicklineColor alone follows with the current behavior in that it represents the color of all the tick lines. That is unless the minor color is set to a value other than automatic.